src/oge/helpers.py

```diff
@@ -194,7 +216,8 @@ def create_plant_ba_table(year):
 
     # specify a ba code for certain utilities
     utility_as_ba_code = pd.read_csv(
-        reference_table_folder("utility_name_ba_code_map.csv")
+        reference_table_folder("utility_name_ba_code_map.csv"),
+        dtype={"name": "string", "ba_code": "string"},
```

Comment:
should we just use column_checks.get_dtypes() here?

Reply:
Same issue, `name` is not in `get_dtypes()`. This is more tricky since a key named name is ambiguous. I would change the name of the header of the first column in the **utility_name_ba_code_map.csv** file to `utility_name`.

Reply:
Probably no need here unless you really want to

Reply:
It can be done in a quick PR later. We have bigger fishes to fry right now.